electronics/_kicad_library/BuckModule.pretty/BuckModule.kicad_mod

```diff
@@ -0,0 +1,28 @@
+(module BuckModule (layer F.Cu) (tedit 594D0F91)
```

Comment:
The other names I came up with for the library itself  are less descriptive; buck because it's a dc-dc converter that lowers voltage, and module because it's on a separate pcb.  I renamed the part though, since it's a 7805 pinout (plus enable)